### PR TITLE
Added WindowOptions type

### PR DESCRIPTION
**Pull request by ColinEberhardt**
I could do with a bit of help documenting the properties with TODOs. They seem quite obvious, but I can't find the electron equivalents.

### CODE REVIEW

File: packages/api-specification/interface.ts [hunk 1]
```diff
@@ -28,7 +127,7 @@ declare namespace ssf {
      * @param {boolean} flag - Flag to start or stop the window flashing.
      * @returns {Promise<void>} A promise which resolves to nothing when the function has completed.
      */
-    flashFrame(flag: boolean): Promise<void>;
+    flashFrame(flag: Number): Promise<void>;
```
Comment (BenLambertNcl) [inline packages/api-specification/interface.ts]: Why has this changed?

File: packages/api-specification/generate-docs.js [hunk 2]
```diff
@@ -70,6 +70,12 @@ const generateInterface = (interfaceObj) => {
 const generateProperty = (prop, className) => {
   const typeArgsString = getType(prop.type);
   mdObject[className] += `**${prop.name}${prop.flags.isOptional ? '?' : ''}**: \`${typeArgsString}\`  \n`;
+  const comment = prop.comment;
+  if (comment) {
+    if (comment.shortText) {
```
Comment (BenLambertNcl) [inline packages/api-specification/generate-docs.js:75]: Could we do `if (comment && comment.shortText)`, or are there other fields we might want in the future?

**Review by BenLambertNcl — COMMENTED** [review]
Looks really good. @ColinEberhardt Do you want me to pull this down and add the descriptions once those comments have been addressed?

File: packages/api-specification/interface.ts (hunk 1 above)
Comment (ColinEberhardt) [reply packages/api-specification/interface.ts]: Ooops! thanks for spotting

File: packages/api-specification/generate-docs.js (hunk 2 above)
Comment (ColinEberhardt) [reply packages/api-specification/generate-docs.js:75]: Yes, although I've got an idea that I think could make this transformation a bit simpler, so I'll give that a go first :-)

**Comment by ColinEberhardt** [comment]
> Looks really good. @ColinEberhardt Do you want me to pull this down and add the descriptions once those comments have been addressed?

Thanks @BenLambertNcl that would be good.

**Comment by BenLambertNcl** [comment]
@ColinEberhardt I have updated the TODOs

**Comment by ColinEberhardt** [comment]
> @ColinEberhardt I have updated the TODOs

Thanks @BenLambertNcl  - I've rebased, so once tests pass, should be good to merge.

**Review by AndyMJones — APPROVED** [review]

**Review by BenLambertNcl — APPROVED** [review]

**Pull request merged**